GameServer/Logic/Maps/SummonersRift.cs

```diff
@@ -15,78 +15,120 @@ namespace LeagueSandbox.GameServer.Logic.Maps
+            new Vector2(1170.0f, 4041.0f),
+            new Vector2(861.0f, 6459.0f),
+            new Vector2(880.0f, 10180.0f),
+            new Vector2(1268.0f, 11675.0f),
+            new Vector2(2806.0f, 13075.0f),
+            new Vector2(3907.0f, 13243.0f),
+            new Vector2(7550.0f, 13407.0f),
+            new Vector2(10244.0f, 13238.0f),
+            new Vector2(10947.0f, 13135.0f),
+            new Vector2(12511.0f, 12776.0f)
+        };
+        private List<Vector2> _blueBotWaypoints = new List<Vector2>
+        {
+            new Vector2(1487.0f, 1302.0f),
+            new Vector2(3789.0f, 1346.0f),
+            new Vector2(6430.0f, 1005.0f),
+            new Vector2(10995.0f, 1234.0f),
+            new Vector2(12841.0f, 3051.0f),
+            new Vector2(13148.0f, 4202.0f),
+            new Vector2(13249.0f, 7884.0f),
+            new Vector2(12886.0f, 10356.0f),
+            new Vector2(12511.0f, 12776.0f)
+        };
+        private List<Vector2> _blueMidWaypoints = new List<Vector2>
+        {
+            new Vector2(1418.0f, 1686.0f),
+            new Vector2(2997.0f, 2781.0f),
+            new Vector2(4472.0f, 4727.0f),
+            new Vector2(8375.0f, 8366.0f),
+            new Vector2(10948.0f, 10821.0f),
+            new Vector2(12511.0f, 12776.0f)
+        };
+        private List<Vector2> _redTopWaypoints = new List<Vector2>
+        {
+            new Vector2(12451.0f, 13217.0f),
+            new Vector2(10947.0f, 13135.0f),
+            new Vector2(10244.0f, 13238.0f),
+            new Vector2(7550.0f, 13407.0f),
+            new Vector2(3907.0f, 13243.0f),
+            new Vector2(2806.0f, 13075.0f),
+            new Vector2(1268.0f, 11675.0f),
+            new Vector2(880.0f, 10180.0f),
+            new Vector2(861.0f, 6459.0f),
+            new Vector2(1170.0f, 4041.0f),
+            new Vector2(1418.0f, 1686.0f)
+        };
+        private List<Vector2> _redBotWaypoints = new List<Vector2>
+        {
+            new Vector2(13062.0f, 12760.0f),
+            new Vector2(12886.0f, 10356.0f),
+            new Vector2(13249.0f, 7884.0f),
+            new Vector2(13148.0f, 4202.0f),
+            new Vector2(12841.0f, 3051.0f),
+            new Vector2(10995.0f, 1234.0f),
+            new Vector2(6430.0f, 1005.0f),
+            new Vector2(3789.0f, 1346.0f),
+            new Vector2(1418.0f, 1686.0f)
+        };
+        private List<Vector2> _redMidWaypoints = new List<Vector2>
+        {
+            new Vector2(12511.0f, 12776.0f),
+            new Vector2(10948.0f, 10821.0f),
+            new Vector2(8375.0f, 8366.0f),
+            new Vector2(4472.0f, 4727.0f),
+            new Vector2(2997.0f, 2781.0f),
+            new Vector2(1418.0f, 1686.0f)
+        };
+
+        private List<string> _regularMinionWave = new List<string>
+        {
+            "MELEE",
+            "MELEE",
+            "MELEE",
+            "CASTER",
+            "CASTER",
+            "CASTER",
+            null,
```

Comment:
Also why is this a `string` list instead of a `MinionSpawnType` list? Would make more sense to use the direct types straight away instead of having to convert.